alfalfa_worker/requirements/requirements.txt

```diff
@@ -0,0 +1,15 @@
+boto3
```

Comment:
I think we should keep the use of poetry. We already have poetry in the root.

Reply:
It would be nice to move to at least  python 3.7 so we can use modern pandas and numpy